code/game/gamemodes/wizard/wizard.dm

```diff
@@ -166,7 +166,7 @@
 
 	for(var/datum/mind/wizard in wizards)
 		if(isliving(wizard.current) && wizard.current.stat!=DEAD)
-			return ..()
+			return 1

```

Comment:
Also this isn't even the correct thing to do. This for loop checks for other living wizards, so you've just told the game to end every wizard round as soon as it starts!